src/chains/ethereum_1/helpers/generalHelpers.ts

```diff
@@ -91,3 +93,15 @@ export function fromTokenValueString(value: string, base: number = 10, precision
   }
   return toTokenValueString(value, base, negativePrecision)
 }
+
+export function matchKnownAbiTypes(contract: EthereumActionContract) {
```

Comment:
@traylewin This was an idea I had for matching ABI types, we can iterate on it or remove it but erc20 transfers wont work unless the exact abi we have is also passed in by the user.

Reply:
If we decide to keep this approach I need to add it to the other templates.